src/components/access-settings/AccessSettings.tsx

```diff
@@ -154,48 +153,16 @@ const AccessSettings: FunctionComponent<{study: Study}> = ({study}) => {
     [id]
   )
 
-  async function createNewAccount(
-    email: string,
-    firstName: string | undefined,
-    lastName: string | undefined,
-    access: Access,
-    token: string,
-    currentUserOrg: string
-  ) {
-    try {
-      let principalId: string | null = null
-      let synapseFirstName: string | null = null
-      let synapseLastName: string | null = null
-
-      if (AccessService.isSynapseEmail(email)) {
-        const synapseAlias = await AccessService.getAliasFromSynapseByEmail(email)
-        principalId = synapseAlias.principalId
-        synapseFirstName = synapseAlias.firstName
-        synapseLastName = synapseAlias.lastName
-      }
-
-      const demoExternalId = await ParticipantService.signUpForAssessmentDemoStudy(token!)
-
-      const {identifier: userId} = await AccessService.createIndividualAccount(
-        token!,
-        email,
-        principalId,
-        firstName || synapseFirstName,
-        lastName || synapseLastName,
-        currentUserOrg,
-        {demoExternalId},
-        getRolesFromAccess(access)
-      )
-
-      if (!AccessService.isSynapseEmail(email)) {
-        await UserService.sendRequestResetPassword(userId, token!)
-      }
-
-      return [true]
-    } catch (error) {
-      return [false, error]
-    }
-  }
+  const {mutate: inviteUser, isLoading: isCreatingNewMember} = useCreateNewAccount({
+    onSuccess: () => {
```

Comment:
`onSuccess` and `onError` replicate the behavior in the old `inviteUser` method, except the `newOrgAccount` is reset to the default template after an account is successfully created, which prevents the last added user's info from pre-populating fields when the add new member form is next opened